lib/internal/Magento/Framework/App/Test/Unit/Console/CommandLoader/AggregateTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Magento\Framework\App\Test\Unit\Console\CommandLoader;
+
+use Magento\Framework\Console\CommandLoader\Aggregate;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\CommandLoader\CommandLoaderInterface;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+
+class AggregateTest extends TestCase
+{
+    private $firstCommandLoader;
+    private $secondCommandLoader;
+    private $subj;
+
+    protected function setUp(): void
+    {
+        $this->firstCommandLoader = $this->getMockBuilder(CommandLoaderInterface::class)->getMock();
+        $this->secondCommandLoader = $this->getMockBuilder(CommandLoaderInterface::class)->getMock();
+        $this->subj = new Aggregate([$this->firstCommandLoader, $this->secondCommandLoader]);
+    }
+
+    /**
+     * @dataProvider provideTestCasesForHas
+     */
+    public function testHas($firstResult, $secondResult, $overallResult)
+    {
+        $this->firstCommandLoader->method('has')->with('foo')->willReturn($firstResult);
+        $this->secondCommandLoader->method('has')->with('foo')->willReturn($secondResult);
+
+        static::assertEquals($overallResult, $this->subj->has('foo'));
```

Comment:
In Magento we generally use `$this->`. It's not a written standard, but for sake of consistency, I'd recommend using the same.